doc/guides/using-internal-errors.md

```diff
@@ -15,9 +15,9 @@ to be breaking changes. By providing a permanent identifier for a specific
 error, we reduce the need for userland code to inspect error messages.
 
 *Note*: Switching an existing error to use the `internal/errors` module must be
-considered a `semver-major` change. However, once using `internal/errors`,
-changes to `internal/errors` error messages will be handled as `semver-minor`
-or `semver-patch`.
+considered a `semver-major` change. In the near future after the transition
+has been completed (but no sooner then `v10`) error message changes will be
```

Comment:
I don't believe we can say "no sooner" than v10. The decision was to handle the changes as semver-major in 8.x and to revisit within 9.x... which means we could start handling them as semver-minor as early as 9.0.0.